overlord/overlord_test.go

```diff
@@ -642,7 +642,7 @@ func (ovs *overlordSuite) TestOverlordStartUpSetsStartOfOperation(c *C) {
 	c.Assert(st.Get("start-of-operation-time", &opTime), IsNil)
 }
 
-func (ovs *overlordSuite) TestEnsureLoopPruneDoesntAbortShortlyAfterStartOfOperation(c *C) {
+func (ovs *overlordSuite) disabledFor244onlyTestEnsureLoopPruneDoesntAbortShortlyAfterStartOfOperation(c *C) {
```

Comment:
```suggestion
func (ovs *overlordSuite) TestEnsureLoopPruneDoesntAbortShortlyAfterStartOfOperation(c *C) {
	// TODO: the test is racy, the fixes did not make it to 2.44
	c.Skip("fixed in master")
```

Reply:
Thanks! Updated with a slightly different doc comment and force-pushed (so that I can cherry pick it easier).